_uw-research-computing/apptainer-htc.md

```diff
@@ -0,0 +1,409 @@
+---
+highlighter: none
+layout: guide
+title: Use Apptainer Containers
+guide:
+    order: 10
+    category: Software Solutions
+    tag:
+        - htc
+--- 
+
+## Introduction
+
+HTCondor supports the use of Apptainer (formerly known as Singularity) environments for jobs on the High Throughput Computing system. 
+
+Similar to Docker containers, Apptainer environments allow users to prepare portable software and computing environments that can be sent to many jobs. 
+This means your jobs will run in a more consistent environment that is easily reproducible by others. 
+
+Container jobs are able to take advantage of more of CHTC's High Throughput resources because the operating system where the job is running does not need to match the operating system where the container was built.
+
+{% capture content %}
+
+[Quickstart](#quickstart)
+
+* [Use an existing container](#use-an-existing-container)
+* [Build your own container](#build-your-own-container)
+
+[Create a Definition File](#create-a-definition-file)
+
+[Start an Interactive Build Job](#start-an-interactive-build-job)
+
+[Build Your Container](#build-your-container)
+
+[Test Your Container](#test-your-container)
+
+[Move the Container .sif File to Staging](#move-the-container-sif-file-to-staging)
+
+[Use an Apptainer Container in HTC Jobs](#use-an-apptainer-container-in-htc-jobs)
+
+{% endcapture %}
+{% include /components/directory.html title="Table of Contents" %}
+
+## Quickstart <a name="quickstart"></a>
+
+### Use an existing container
```

Comment:
on further reflection...I might move this to the end and in the "build a container" section, mention that if you already HAVE a container, to skip down. Maybe? 

Let's discuss.